### DISCUSSION TITLE
Why My Flet App Not Running On  repl.it?

### DISCUSSION THREAD

**Discussion by ArnavK-09** [flet-dev/flet#1189] [Q&A]
![Screenshot_2023-03-17-22-08-20-51_dce875ef40efa4e902b2719365b6f678.jpg](https://user-images.githubusercontent.com/69188140/225966066-8c338825-171e-4dee-a412-e98456cbb84e.jpg)

It just shows me log and no output

My app.py:

```py

# imports
import flet as ft
from views import views_handler


def main(page: ft.Page):

    # route handler
    def handle_route(_):
        """
          Matches the given route with view_handler
        """

        page.views.clear()
        try:
            page.views.append(
                views_handler(page)[page.route]
            )
        except:
            page.views.append(
                views_handler(page)['/']
            )
            
    # customizations
    page.window_bgcolor = ft.colors.BLACK
    page.bgcolor = ft.colors.BLACK

    appbar_items = [
        ft.PopupMenuItem(text="Login"),
        ft.PopupMenuItem(),  # divider
        ft.PopupMenuItem(text="Settings")
    ]
    appbar = ft.AppBar(
            leading=ft.Icon(ft.icons.GRID_GOLDENRATIO_ROUNDED),
            leading_width=100,
            title=ft.Text("Trolli",size=32, text_align="start"),
            center_title=False,
            toolbar_height=75,
            bgcolor=ft.colors.LIGHT_BLUE_ACCENT_700,
            actions=[
                ft.Container(
                    content=ft.PopupMenuButton(
                        items=appbar_items
                    ),
                    margin=ft.margin.only(left=50, right=25)
                )
            ],
    )
    page.appbar = appbar
    page.update()

    # on route change
    page.on_route_change = handle_route
    # land on index page
    page.go('/')

print("started")
# start
ft.app(target=main,web_renderer="html")


```

Also, if i use 
```py
# start
ft.app(target=main, view=ft.WEB_BROWSER)
```

It gives me this err:

![Screenshot_2023-03-17-22-18-53-27_dce875ef40efa4e902b2719365b6f678.jpg](https://user-images.githubusercontent.com/69188140/225967967-1b4753cc-4623-4d2a-851f-0c6cf6bf8664.jpg)

Edit:
If i run - flet publish

And run dist

It shows infinite loading

![Screenshot_2023-03-17-22-33-28-77_dce875ef40efa4e902b2719365b6f678.jpg](https://user-images.githubusercontent.com/69188140/225971395-d70bd326-30a9-4cf5-b070-2b36906825f6.jpg)

HELLP

**Comment by FeodorFitsner** [comment]
Flet is currently not working on mobile, sorry.

**Reply by ArnavK-09** [reply]
But I'm using repl, also it is not working on repl.it browser 

Btw working on GitHub codespaces

**Comment by ndonkoHenri** [comment]
This problem has been mentioned several times on discord @FeodorFitsner.
Try it by creating a new repl and running a simple hello code.

**Reply by FeodorFitsner** [reply]
Sorry, didn't notice it's about replit. Will take a look what's wrong with that.

**Reply by ArnavK-09** [reply]
Same errors as before

![Screenshot_2023-03-18-19-16-39-90_dce875ef40efa4e902b2719365b6f678.jpg](https://user-images.githubusercontent.com/69188140/226110041-7ec08d67-d37b-4cb0-861a-e406fc3f37dc.jpg)

![Screenshot_2023-03-18-19-17-05-47_dce875ef40efa4e902b2719365b6f678.jpg](https://user-images.githubusercontent.com/69188140/226110045-656190d4-3a1f-4465-bdbc-2a12d371d435.jpg)




Code:
```py
import flet as ft

def main(page: ft.Page):
    t = ft.Text(value="Hello, world!", color="green")
    page.controls.append(t)
    page.update()

ft.app(target=main)
```
Or
```py
ft.app(target=main, view=ft.WEB_BROWSER)
```
Or
```py
ft.app(target=main, web_renderer="html")
```

🥲

Is there any other platform i can edit on mobile for sometime
